src/UnigramWordPredictor.java

```diff
@@ -50,8 +51,22 @@ public UnigramWordPredictor(Tokenizer tokenizer) {
    */
   public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
+    Map<String, List<String>> wordMap = new HashMap<>();
```

Comment:
This works, but you could consider just using `neighborMap` directly instead of creating a new local variable and then assigning it to `neighborMap`.